pkg/api/response/response.go

```diff
@@ -187,6 +188,26 @@ func JSONStreaming(status int, body interface{}) StreamingResponse {
 	}
 }
 
+// JSONDownload creates a JSON response indicating that it should be downloaded.
+func JSONDownload(status int, body interface{}, filename string) *NormalResponse {
+	return JSON(status, body).SetHeader("Content-Disposition", fmt.Sprintf(`attachment;filename="%s"`, filename))
```

Comment:
Feel free to indent these if you think it might improve readability

```suggestion
	return JSON(status, body).
		SetHeader("Content-Disposition", fmt.Sprintf(`attachment;filename="%s"`, filename))
```

Reply:
Fixed b68338c787c7dc807a33b6635c6ad130e7dcf3dd